indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -149,14 +149,18 @@
             var $breadcrumbs = $('<ul>', {class: 'breadcrumbs'});
             var tag = clickable ? '<a>' : '<span>';
 
-            _.each(path, function(category) {
-                var $item = $(tag, {text: category.title});
+            _.each(path, function(category, idx) {
+                var $item = $('<li>');
```

Comment:
It's not the prettiest, I agree, but if we go for some more semantic HTML and having `"in "` outside of the `<ul>` then we have to wrap the whole `<ul>` and make that The Breadcrumbs. It's not a big change in any case if you want to go for it.